playground/graph/graph_examples.ml

```diff
@@ -0,0 +1,47 @@
+open Graph
+open Node
+
+(* #use "graph_new_examples.ml";; *)
+let () = Printf.printf "Creating nodes with values 4, 5, 6...\n"
+let node1 = Node.create 4
+let node2 = Node.create 5
+let node3 = Node.create 6
+
+
+let () = Printf.printf "Creating graph with nodes: {node1, node2, node3} and creating edges node1-node2 and node2-node3...\n"
+let my_graph = 
+  Graph.empty true
+  |> Graph.add_node node1 
+  |> Graph.add_node node2 
+  |> Graph.add_node node3
+  |> Graph.add_edge node1 node2
+  |> Graph.add_edge node2 node3
+
+
+let () = Printf.printf "Printing graph to string...\n"
```

Comment:
To se da krajše napisati kot `print_endline "..."`

Reply:
2730adeb273067d1eb6a882be53c2797132e0c21